src/client/Client.js

```diff
@@ -359,6 +378,11 @@ class Client extends BaseClient {
    * Obtains the OAuth Application of the bot from Discord.
    * @param {Snowflake} [id='@me'] ID of application to fetch
    * @returns {Promise<ClientApplication>}
+   * @example
+   * client.fetchApplication('id')
+   *  .then(application => {
+   *    console.log('Obtained application with name: ${application.name}');
```

Comment:
Just one thing, this should use template literals.

Reply:
👀 whoops